### ISSUE TITLE
support streaming messages

### ISSUE THREAD

**Issue by bruffridge** [nasa-petal/bidara-deep-chat#73]
I think support for the new Assistants API streaming was added to deep-chat.

https://github.com/OvidijusParsiunas/deep-chat/issues/137

**Comment by jackitaliano** [comment]
Yes, it was, but not there. That is simulating streaming. It still doesn't return anything until the whole message is received, but it then slowly "streams" the content. Results actually in a longer wait time.

However, actual streaming was added, but only in dev so far. 
https://github.com/OvidijusParsiunas/deep-chat/issues/155

Should be in a coming update, but will also result in some other changes that should be addressed due to deep-chat changes.

**Comment by jackitaliano** [comment]
Deep-chat maintainer is working on a solution to issue I mentioned. He said it would be non-trivial, so I'll keep tabs on that.

https://github.com/OvidijusParsiunas/deep-chat/issues/157

**Comment by jackitaliano** [comment]
This issue seems to be resolved in `deep-chat-dev` `v9.0.164`; however, some other issues remain as mentioned #88, so I'll wait to put that in.

**Comment by bruffridge** [comment]
I was working on this a little, and one issue I encountered is I wasn't able to get the `responseInterceptor` to fix the image links since the function is called once for each token returned from streaming, so couldn't use the existing find and replace code. So moving on for now. I'm not sure how valuable streaming is anyways other than slightly better UX.

**Comment by jackitaliano** [comment]
Have fixed some of the issues with streaming on our end; however, one new bug presented itself on `deep-chat`'s. Have an issue opened, but don't expect it to be resolved for another week due to the maintainer being OOO.
